usaspending_api/disaster/v2/views/agency/count.py

```diff
@@ -1,34 +1,48 @@
-from django.db.models import OuterRef, Q, Exists
+from django.db.models import OuterRef, Q, Exists, Count
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from usaspending_api.common.cache_decorator import cache_response
 from usaspending_api.disaster.v2.views.count_base import CountBase
+from usaspending_api.awards.models import FinancialAccountsByAwards
 from usaspending_api.financial_activities.models import FinancialAccountsByProgramActivityObjectClass
 from usaspending_api.references.models import ToptierAgency
+from usaspending_api.disaster.v2.views.disaster_base import AwardTypeMixin
 
 
-class AgencyCountViewSet(CountBase):
+class AgencyCountViewSet(CountBase, AwardTypeMixin):
```

Comment:
If using this mixin you should order it as `AgencyCountViewSet(AwardTypeMixin, CountBase)` and then you can remove the `required_filters` below since the mixin will override `required_filters` in DisasterBase.